lib/workers/repository/init/inherited.ts

```diff
@@ -99,5 +101,42 @@ export async function mergeInheritedConfig(
       'Removed global config from inherited config.',
     );
   }
-  return mergeChildConfig(config, filteredConfig);
+
+  if (is.nullOrUndefined(filteredConfig.extends)) {
+    return mergeChildConfig(config, filteredConfig);
+  }
+
+  let returnConfig = filteredConfig;
+
+  // Decrypt after resolving, in case the preset contains npm authentication
+  logger.debug('Resolving presets found in inherited config');
+  const resolvedConfig = await decryptConfig(
+    await resolveConfigPresets(returnConfig, config, config.ignorePresets),
+    config.repository,
+  );
```

Comment:
npm presets have been deprecated for a long time, please remove this logic to simplify things